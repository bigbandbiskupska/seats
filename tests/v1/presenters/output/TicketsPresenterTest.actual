[
	'id' => 1,
	'user_id' => 1,
	'note' => '007',
	'created_at' => Nette\Utils\DateTime::__set_state(/* #2c14 */ [
		'date' => '2017-01-01 00:00:00.000000',
		'timezone_type' => 3,
		'timezone' => 'Europe/Prague',
	]),
	'updated_at' => Nette\Utils\DateTime::__set_state(/* #7294 */ [
		'date' => '2017-01-01 00:00:00.000000',
		'timezone_type' => 3,
		'timezone' => 'Europe/Prague',
	]),
	'seats' => [
		[
			'id' => 1,
			'schema_id' => 1,
			'x' => 1,
			'y' => 1,
			'row' => 1,
			'col' => 1,
		],
		[
			'id' => 2,
			'schema_id' => 1,
			'x' => 1,
			'y' => 2,
			'row' => 1,
			'col' => 2,
		],
		[
			'id' => 3,
			'schema_id' => 1,
			'x' => 1,
			'y' => 3,
			'row' => 1,
			'col' => 3,
		],
		[
			'id' => 4,
			'schema_id' => 1,
			'x' => 1,
			'y' => 4,
			'row' => 1,
			'col' => 4,
		],
		[
			'id' => 5,
			'schema_id' => 1,
			'x' => 1,
			'y' => 5,
			'row' => 1,
			'col' => 5,
		],
	],
]
