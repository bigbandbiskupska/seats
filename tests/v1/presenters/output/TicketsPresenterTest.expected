[
	'id' => 1,
	'user_id' => 1,
	'note' => '007',
	'seats' => [
		[
			'id' => 1,
			'schema_id' => 1,
			'x' => 1,
			'y' => 1,
			'row' => 1,
			'col' => 1,
			'price' => 250,
		],
		[
			'id' => 2,
			'schema_id' => 1,
			'x' => 1,
			'y' => 2,
			'row' => 1,
			'col' => 2,
			'price' => 250,
		],
		[
			'id' => 3,
			'schema_id' => 1,
			'x' => 1,
			'y' => 3,
			'row' => 1,
			'col' => 3,
			'price' => 250,
		],
		[
			'id' => 4,
			'schema_id' => 1,
			'x' => 1,
			'y' => 4,
			'row' => 1,
			'col' => 4,
			'price' => 250,
		],
		[
			'id' => 5,
			'schema_id' => 1,
			'x' => 1,
			'y' => 5,
			'row' => 1,
			'col' => 5,
			'price' => 250,
		],
	],
	'created_at' => Nette\Utils\DateTime::__set_state(/* #8308 */ [
		'date' => '2017-01-01 00:00:00.000000',
		'timezone_type' => 3,
		'timezone' => 'Europe/Prague',
	]),
	'updated_at' => Nette\Utils\DateTime::__set_state(/* #d7e3 */ [
		'date' => '2017-01-01 00:00:00.000000',
		'timezone_type' => 3,
		'timezone' => 'Europe/Prague',
	]),
]
